executor/http_runner.go

```diff
@@ -162,11 +161,14 @@ func (f *HTTPFunctionRunner) Run(req FunctionRequest, contentLength int64, r *ht
 	if res.Body != nil {
 		defer res.Body.Close()
 
-		bodyBytes, bodyErr := ioutil.ReadAll(res.Body)
+		bodyBytes, bodyErr := io.ReadAll(res.Body)
 		if bodyErr != nil {
 			log.Println("read body err", bodyErr)
 		}
-		w.Write(bodyBytes)
+
+		if _, err = w.Write(bodyBytes); err != nil {
```

Comment:
Not sure this is something we want to / need to log and handle.

Reply:
will be reverted